std/conv.d

```diff
@@ -27,7 +27,7 @@ import std.range.primitives;
 import std.traits;
 import std.typetuple;
 
-private string convFormat(Char, Args...)(in Char[] fmt, Args args)
+private S convFormat(S, Args...)(in S fmt, Args args) if (isSomeString!S)
```

Comment:
Do we even call it internally anywhere? It's private should be easy to just drop it...